### PR TITLE
Add tooltips for other stacked graphs

### PR DESCRIPTION
**Pull request by DanielMwendwa**
copied code from PR #182 and updated the variable according to the graph the tooltip is being applied and the message direction

### CODE REVIEW

File: public/graph.js [hunk 1]
```diff
@@ -361,6 +361,37 @@ class GraphController {
                 )
                 .attr("width", Width / Object.keys(dataFilteredWeek).length);
 
+            // Add tooltip for the total received sms graph
```
Comment (IsaackMwenda) [inline public/graph.js]: We don't need tooltips for 10 mins view let us have it in per day only and add this if requested by the team.

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]

File: public/graph.js (hunk 1 above)
Comment (DanielMwendwa) [reply public/graph.js]: removed tooltips for 10 minute view

Comment (IsaackMwenda) [reply public/graph.js]: Thanks

**Review by IsaackMwenda — APPROVED** [review]
Thanks, @DanielMwendwa!

**Pull request merged**